### DISCUSSION TITLE
Compatibility with Atlantic Hybea

### DISCUSSION THREAD

**Discussion by Syxxxx** [fredlcore/BSB-LAN#684] [Q&A] [answered]
Hello,
BSB-lan work with Atlantic Hybea https://www.atlantic.fr/Chauffer-le-logement/Pompe-a-chaleur/Pompe-a-chaleur-hybride/Hybea ?
Thanks

**Comment by fredlcore** [comment, marked as answer]
No, unfortunately, it doesn't seem to be compatible.

**Reply by Syxxxx** [reply]
Thanks for your reply.
Is there anything I can do to help with future support (photo) ?

**Reply by fredlcore** [reply]
No, unfortunately not. BSB-LAN works on a special bus (BSB/LPB/PPS), and if the device doesn't have that bus, it won't even work on a hardware level with our adapter.
